assets/src/edit-story/components/canvas/singleSelectionMovable.js

```diff
@@ -245,6 +250,21 @@ function SingleSelectionMovable({ selectedElement, targetEl, pushEvent }) {
         if (updates && updates.height) {
           newHeight = dataToEditorY(updates.height);
         }
+
+        if (isResizingFromCorner) {
```

Comment:
I think this code should go above, before we try to run `updateForResizeEvent`. Otherwise the `font-size` and other adjustments will all be calculated from the wrong values.